index.html

```diff
@@ -0,0 +1,64 @@
+<!-- landing page for personal portfolio -->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title> Tamiko Terada </title>
+    <link rel="stylesheet" type="text/css" href="stylesheets/main.css">
+    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css">
+  </head>
+
+  <body>
+    <a href="html/menu.html" target="_parent"><button autofocus id="test"> </button></a>
+    <!-- header | name, image, banner, button-link -->
+    <header>
+      <div class="left-bar">
+        <p class="top-bar">Tamiko Terada</p>
```

Comment:
Semantic HTML: Is `<p class="top-bar">Tamiko Terada</p>` really a paragraph?

Also, it might be nice to make this a link to either your LinkedIn or your About page.